charts/perses/values.schema.json

```diff
@@ -102,6 +102,14 @@
               "type": "boolean",
               "default": false
             },
+            "encryptionKey": {
```

Comment:
I think makes sense add this values to the values.yaml as empty values, just to let users know it exists and is a possibility, as well as because the documentation is being created from the values file.